.travis.yml

```diff
@@ -9,11 +9,13 @@ env:
 before_install:
   - brew update 
   - brew install carthage
+
+install:
   - carthage bootstrap --use-submodules --no-build
+  - pod repo update --silent
```

Comment:
Yes, this is still necessary because lint fails if repo doesn't have specified podspec of Result.